ref_ts/templates/pages/reftimeseriesresource.html

```diff
@@ -1,114 +1,107 @@
         </span>
-     </a>
+    </a>
 {% endblock download_bag %}
 
 {% block extended_metadata %}
     {% if extended_metadata_exists %}
-        <div class="row">
-            <div class="col-md-12">
-                <div>
-                    {% if referenceURL %}
-                        <h4><strong>Reference URL:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        <div class="row">
-                            <div class="col-sm-4"><strong>URL:</strong></div>
-                            <div class="col-sm-8">{{ referenceURL.value }}</div>
-                        </div>
-                        <div class="row">
-                            <div class="col-sm-4"><strong>Type:</strong></div>
-                            <div class="col-sm-8">{{ referenceURL.type }}</div>
-                        </div>
-                    <hr style="border:0">
-                    {%  endif %}
+        <div class="col-md-12">
+            {% if referenceURL %}
+                <legend>Reference URL</legend>
+                <div class="space-bottom">
+                    <div class="row">
+                        <div class="col-sm-4"><strong>URL:</strong></div>
+                        <div class="col-sm-8">{{ referenceURL.value }}</div>
+                    </div>
+                    <div class="row space-bottom">
+                        <div class="col-sm-4"><strong>Type:</strong></div>
+                        <div class="col-sm-8">{{ referenceURL.type }}</div>
+                    </div>
                 </div>
-                <div>
-                    {% if site and site.code or site and site.name or site and site.net_work %}
-                        <h4><strong>Site:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        {% if site.code %}
+            {% endif %}
+
+            {% if site and site.code or site and site.name or site and site.net_work %}
+                 <legend>Site</legend>
+                <div class="space-bottom">
+                    {% if site.code %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Code:</strong></div>
                             <div class="col-sm-8">{{ site.code }}</div>
                         </div>
-                        {%  endif %}
-                        {% if site.name %}
+                    {% endif %}
+                    {% if site.name %}
                         <div class="row">
                             <div class="col-sm-4"><strong>Name:</strong></div>
                             <div class="col-sm-8">{{ site.name }}</div>
                         </div>
-                        {%  endif %}
-                        {% if site.net_work %}
+                    {% endif %}
+                    {% if site.net_work %}
```

Comment:
I noticed you made indentation change here.

Reply:
Correct. The indentation we had before was off.